Gemfile

```diff
@@ -2,8 +2,9 @@ source "https://rubygems.org"
 gemspec
 
 gem "rails", "~> 3.2.12"
-gem "sqlite3-ruby", require: "sqlite3"
-gem "minitest", '~> 4.7', :platforms => :ruby_19
+gem "sqlite3"
```

Comment:
```
Post-install message from sqlite3-ruby:

#######################################################

Hello! The sqlite3-ruby gem has changed it's name to just sqlite3.  Rather than
installing `sqlite3-ruby`, you should install `sqlite3`.  Please update your
dependencies accordingly.

Thanks from the Ruby sqlite3 team!

<3 <3 <3 <3

#######################################################
```